cleanlab/filter.py

```diff
@@ -446,7 +446,11 @@ class 0, 1, ..., K-1.
     if filter_by not in ["low_self_confidence", "low_normalized_margin"]:
         # Remove label issues if given label == model prediction if issues haven't been removed yet
         pred = pred_probs.argmax(axis=1)
-        label_issues_mask[pred == labels] = False
+        mask = pred == labels
+        if K == 2:
+            # Set mask to False wherever pred_probs == 0.5 where num_classes == 2
+            mask = mask & (pred_probs[:, 0] != 0.5)
```

Comment:
Should this be between +/- (0.5 + TINY_VALUE)?

Reply:
yes it should be:

pred_probs[:,given_label] > 0.5 - EPS